01/app.js

```diff
@@ -6,9 +6,10 @@ function getProductList() {
 }
 
 function getTotalPrice(productList) {
-    return productList.reduce( (product, acc) => {
-        return acc + (product.price * product.count);
-    });
+    return productList.reduce( (acc, product)=> {
+        acc += (product.price * product.count);
```

Comment:
Zwróć uwagę, że jeszcze mamy `discount` - to należy uwzględnić w cenie.

PS. Pamiętaj o zaokrągleniu :)